AutoUpdater.lua

```diff
@@ -6,19 +6,19 @@ function checkForUpdate()
     WebRequest.get('yourgithubfile.json', function(res)
         if (not(res.is_error)) then
             local response = JSON.decode(res.text)
-		if (response[ScriptClass] > ScriptVersion) then
+		if (response[ScriptVersion] > ScriptVersion) then
```

Comment:
We were using the `ScriptClass` as a key to find the latest version - using a version number doesn't really make sense

Reply:
It seems very counterintuitive to have both `ScriptClass` and `ScriptVersion` in the script and then use the `ScriptClass` tag to track the version in the json.

But if you think using a version number to find the last version doesn't make sense, I don't know what makes sense to you...

Reply:
response[ScriptClass] is correct because the GET returns a global manifest with ALL my scripts rather than a single entry.

{"mini.controller":"2.0.3","mini.injector":"3.2.0"} - ScriptClass == "mini.controller" or "mini.injector" in the context of this manifest JSON.